sources/OpenMcdf.Extensions/OLEProperties/PropertySetStream.cs

```diff
@@ -214,8 +214,8 @@ public void Write(System.IO.BinaryWriter bw)
             {
                 for (int i = 0; i < PropertySet1.PropertyIdentifierAndOffsets.Count; i++)
                 {
-                    bw.Seek((int)oc1.PropertyIdentifierOffsets[i], System.IO.SeekOrigin.Begin); //Offset of 4 to Offset value
-                    bw.Write(oc1.PropertyOffsets[i] - oc1.OffsetPS);
+                    bw.Seek((int)oc1.PropertyIdentifierOffsets[i] + 4, System.IO.SeekOrigin.Begin); //Offset of 4 to Offset value
+                    bw.Write((int)(oc1.PropertyOffsets[i] - oc1.OffsetPS));

```

Comment:
Changed this to match the case on lines 207/208, otherwise it was writing the offset in the wrong place, and also writing 8 bytes rather than 4